OpenRA.Mods.Common/Widgets/Logic/Lobby/LobbyUtils.cs

```diff
@@ -562,6 +581,29 @@ public static void SetupTeamWidget(Widget parent, Session.Slot s, Session.Client
 			HideChildWidget(parent, "TEAM_DROPDOWN");
 		}
 
+		public static void SetupEditableHandicapWidget(Widget parent, Session.Slot s, Session.Client c, OrderManager orderManager, MapPreview map)
+		{
+			var dropdown = parent.Get<DropDownButtonWidget>("HANDICAP_DROPDOWN");
```

Comment:
How are we documenting, if at all, (newly) required widgets?

Reply:
https://github.com/OpenRA/OpenRAModSDK/wiki/Update-notes:-Release-20200503-to-next-release, which still needs to be completed before the next playtest.